web/app/webpack.config.js

```diff
@@ -24,7 +24,13 @@ module.exports = {
         exclude: /node_modules/,
         use: [
           'babel-loader',
-          { loader: 'eslint-loader', options: { fix: true } }
+          {
+            loader: 'eslint-loader',
+            options: {
+              fix: true,
+              emitWarning: process.env.NODE_ENV !== 'production'
```

Comment:
Relating to @grampelberg's comment:

> I'd like to make sure CI breaks when eslint is failing.

Our CI configuration runs tests with `bin/web test`, without setting `NODE_ENV`. So I think `NODE_ENV` defaults to 'development' when running tests in CI, in which case branches with lint errors won't actually fail the CI run. Maybe there's a different way to configure this to ensure that CI catches lint errors?

Reply:
Looks like the build sets it correctly and testing doesn't modify from the default (development). Could change it to `process.env.NODE_ENV === 'development'` and then set `NODE_ENV=testing` in `.travis.yml`.

Reply:
I've added a `NODE_ENV` to the `travis.yml` and committed a lint error. If CI fails on this commit we'll know everything works.

Reply:
Eugh,CI still passes. Looks like `bin/web` doesn't exit even if the `build` function returns a non-zero exit code. Looking into it now.